CHANGELOG.md

```diff
@@ -65,6 +65,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 - Added `--traffic_ops_insecure=<0|1>` optional option to traffic_ops_ort.pl
 - Added User-Agent string to Traffic Router log output.
 - Added default sort logic to GET API calls using Read()
+- Added locationByDeepCoverageZone to the `crs/stats/ip/{ip}` endpoint in the Traffic Router API
```

Comment:
@mattjackson220 This got added to the 5.0.0 changelog entries, not the *unreleased* changelog entries.